tests/main/interfaces-udisks2/task.yaml

```diff
@@ -4,22 +4,18 @@ details: |
     The udisks2 interface allows operating as or interacting with the UDisks2 service
 
 # Interfaces not defined for ubuntu core systems
-# FIXME: `udisksctl mount -b  "$device"` fails on arch with:
-#   Object /org/freedesktop/UDisks2/block_devices/loop200 is not a mountable filesystem.
 # 2021-11-02: disabled ubuntu-18.04-32 because it keeps failing when trying
 #             to create/use /dev/loop200
-systems: [-ubuntu-core-*, -arch-linux-*, -centos-9-*, -fedora-*, -ubuntu-18.04-32]
+systems: [-ubuntu-core-*, -ubuntu-18.04-32]
 
 environment:
     FS_PATH: "$(pwd)/dev0-fake0"
-    MMCBLK_PATH: /dev/mmcblk-fake0
 
 prepare: |
     snap install test-snapd-udisks2
 
 restore: |
-    losetup -d "$MMCBLK_PATH" || true
-    rm -f "$MMCBLK_PATH" "$FS_PATH"
+    rm -f "$FS_PATH"
```

Comment:
I guess we need to call `udisksctl loop-delete` here?

Reply:
makes sense, I'll update that

thanks for the review

Reply:
@mardy, I updated that, also removed 'rm -f "$FS_PATH"' as it is not neded because $(pwd) is auto cleaned after each test

Reply:
Thanks!